content/rc/administration/setup/create-subscription.md

```diff
@@ -25,22 +27,21 @@ To create a new subscription:
 1. In the Redis Cloud menu, click **Subscriptions**.
 1. At the bottom of the page, click ![Add](/images/rs/icon_add.png#no-click "Add").
 1. Select your subscription configuration:
+
     1. Select a cloud provider: **Amazon AWS**, **Microsoft Azure**, **Google Cloud Platform**
-    1. Select the region that you want the subscription to use, for example: `us-central1`
-    1. In the Redis Cloud service levels, select the memory or throughput limit for your subscription:
-        - Essentials - You can also select either [Replication]({{< relref "/rc/concepts/clustering.md" >}}) or Multi-Availability Zone (Multi-AZ)
-        - Pro - You can also select a subscription with [Redis on Flash (RoF)]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}}) support
+    1. Select the region that you want the subscription to use, for example: `us-central1`.
+    1. In the Redis Cloud service levels, select the memory or throughput limit for your subscription.  To learn about your options, see [Cloud Pricing](https://redislabs.com/redis-enterprise-cloud/pricing/).
+<!--
+        - Fixed - You can also select either [Replication]({{< relref "/rc/concepts/clustering.md" >}}) or Multi-Availability Zone (Multi-AZ)
+        - Flexible - You can also select a subscription with [Redis on Flash (RoF)]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}}) support
         - Customize - You can:
             - [**Build a Plan**]({{< relref "/rc/administration/setup/customize-pro.md" >}}) to follow a sizing process where you tell us your requirements and we give you a subscription that's just right for you.
             - **Request Pricing** to customize an Ultimate subscription that includes annual payments, premium support, and customer success packages.
-1. After you select a subscription configuration:
-    1. Review the subscription configuration.
+-->
+4. After you select a subscription configuration, update technical details as requested.
```

Comment:
What do you mean by "as requested" here? Requested by whom?

Reply:
Reworked.